test/components/sortable-densities.test.tsx

```diff
@@ -18,16 +19,16 @@ describe("SortableDensities component", () => {
   it("respects `densityWordInPlanetWizard` config option", () => {
     config.densityWordInPlanetWizard = true;
 
-    let wrapper = shallow(<SortableDensities simulationStore={store} />);
-    expect(wrapper.html()).toEqual(expect.stringContaining("LOW"));
-    expect(wrapper.html()).toEqual(expect.stringContaining("HIGH"));
-    expect(wrapper.html()).toEqual(expect.stringContaining("Click and drag to reorder the plate density"));
+    render(<SortableDensities simulationStore={store} />);
+    expect(screen.getByText("LOW")).toBeInTheDocument();
+    expect(screen.queryByText("HIGH")).toBeInTheDocument();
+    expect(screen.queryByText("Click and drag to reorder the plate density")).toBeInTheDocument();
 
     config.densityWordInPlanetWizard = false;
 
-    wrapper = shallow(<SortableDensities simulationStore={store} />);
-    expect(wrapper.html()).toEqual(expect.stringContaining("BELOW"));
-    expect(wrapper.html()).toEqual(expect.stringContaining("ABOVE"));
-    expect(wrapper.html()).toEqual(expect.stringContaining("Click and drag to reorder the plates"));
+    render(<SortableDensities simulationStore={store} />);
+    expect(screen.queryByText("BELOW")).toBeInTheDocument();
+    expect(screen.queryByText("ABOVE")).toBeInTheDocument();
+    expect(screen.queryByText("Click and drag to reorder the plates")).toBeInTheDocument();
```

Comment:
As mentioned above, I think `getByText()` would be more conventional for all of these.

Reply:
Done.